src/Bindings/ManualBindings_World.cpp

```diff
@@ -1483,6 +1483,64 @@ static int tolua_cWorld_SetBlock(lua_State * tolua_S)
 
 
 
+static int tolua_cWorld_SetBlockMeta(lua_State * tolua_S)
+{
+	/* Function signature:
+	World:SetBlockMeta(BlockX, BlockY, BlockZ, BlockMeta)
+	--or--
+	World:SetBlockMeta(Position, BlockMeta)
+	*/
+
+	cLuaState L(tolua_S);
+	int OffsetIndex;
+	if (
+		!L.CheckParamSelf("cWorld") ||
+		!CheckParamVectorOr3Numbers(L, "Vector3<int>", 2, OffsetIndex) ||
+		!L.CheckParamNumber(OffsetIndex) ||
+		!L.CheckParamEnd(OffsetIndex + 1)
+	)
+	{
+		return 0;
+	}
+
+	if (OffsetIndex != 3)  // Not the vector overload
+	{
+		L.LogStackTrace();
+		LOGWARN("SetBlockMeta with 3 position arguments is deprecated, use vector-parametered version instead.");
```

Comment:
Can we use the Manual-Binding Log? So that we include the plugin name?

Reply:
LogFromLuaStack? That takes plugin supplied arguments from the Lua stack. The LogStackTrace prints the plugin directory:

```
[504d3653|20:41:19] Stack trace:
[504d3653|20:41:19]   [C](-1): SetBlockMeta
[504d3653|20:41:19]   Plugins\ChatLog/plugin.lua(29): (no name)
[504d3653|20:41:19] Stack trace end
[504d3653|20:41:19] SetBlockMeta with 3 position arguments is deprecated, use vector-parametered version instead.
```